.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -16,7 +16,7 @@ v    Please also ensure that this is not a duplicate issue :)
 
 ## Version
 
-<!-- git commit hash, output of `gaiacli --version`, output of `gaiad --version` -->
+<!-- SemVer release number, output of `gaiacli version`, output of `gaiad version` -->
```

Comment:
I think we should leave `git commit hash` because the error could be on `master` and not a release, in which case there is no semver.

Reply:
updated :+1: